charts/brigade-metrics/templates/prometheus/configmap.yaml

```diff
@@ -17,7 +17,7 @@ data:
     - job_name: 'node-exporter'
 
       # Override the global default and scrape targets from this job every 5 seconds.
-      scrape_interval: {{ .Values.prometheus.scrapeInterval }}s
+      scrape_interval: {{ .Values.prometheus.scrapeInterval }}
```

Comment:
Don't need the `s` hardcoded here anymore. Units like `s` or `m` can be included right in the `values.yaml`. Both the exporter can properly interpret that now.